finder/tagged.go

```diff
@@ -175,24 +190,31 @@ func TaggedTermWhereN(term *TaggedTerm) (string, error) {
 			// container_name!=""  ==> container_name exists and it is not empty
 			return fmt.Sprintf("arrayExists((x) -> %s, Tags)", where.HasPrefixAndNotEq("x", term.Key+"=")), nil
 		}
+		whereLikeAnyVal := where.HasPrefix("x", term.Key+"=")
 		if strings.Index(term.Value, "*") >= 0 {
-			return fmt.Sprintf("NOT arrayExists((x) -> %s, Tags)", where.Like("x", term.concatMask())), nil
+			whereLike := where.Like("x", term.concatMask())
+			return fmt.Sprintf("arrayExists((x) -> %s, Tags) AND NOT arrayExists((x) -> %s, Tags)", whereLikeAnyVal, whereLike), nil
```

Comment:
Checks if a tag in a 'not equal' term exists at all for a given name. **This is different from how carbon does this.** Carbon doesn't check this.

Reply:
Now only with `dont-match-missing-tags=true`.